roomserver/api/wrapper.go

```diff
@@ -42,10 +42,11 @@ func SendEvents(
 
 // SendEventWithState writes an event with KindNew to the roomserver
 // with the state at the event as KindOutlier before it. Will not send any event that is
-// marked as `true` in haveEventIDs
+// marked as `true` in haveEventIDs.
 func SendEventWithState(
 	ctx context.Context, rsAPI RoomserverInternalAPI, state *gomatrixserverlib.RespState,
 	event gomatrixserverlib.HeaderedEvent, haveEventIDs map[string]bool,
+	roomVersion gomatrixserverlib.RoomVersion,
```

Comment:
What's the reason for this change? The `event` (as a `HeaderedEvent`) should contain the room version already, and by being in the same room, all of the outliers are going to be the same room version as the title event.

Reply:
oops, this was an accidental leftover from something